arch/arm/dts/imx8mm-evk-u-boot.dtsi

```diff
@@ -181,4 +189,4 @@
 	/delete-property/ assigned-clocks;
 	/delete-property/ assigned-clock-parents;
 	/delete-property/ assigned-clock-rates;
-};
```

Comment:
This change is not needed?

Reply:
Shouldn't be, done automatically by vim (adding end of the line), let me see if I can drop this piece.

Reply:
Updated.